scala/README.md

```diff
@@ -145,7 +156,7 @@ Please read the [Scala Style Guide] carefully. The main points to consider are:
 
     Inside each group, packages and classes must be sorted alphabetically. Separate groups with blank lines.
 
-    Use the following mnemonic to remember package order: your project is built on top of the frameworks, which is built on Scala, which is built on top of Java. Third-party libraries are less important, therefore come last.
+    Use the following mnemonic to remember package order: your project is built on top of the frameworks, which are built on Scala, which is built on top of Java. Third-party libraries are less important, therefore come last.
```

Comment:
not sure if this is technically a mnemonic. also, not sure if we agree on this one @jarhart @chrizkim ? on echo/newaer we've been doing:
```
import my.package.models

import org.scalatest._

import scala.concurrent.Future
import java.util.whatever
```
i guess i don't care that much, but curious what others think so we can settle on this